packages/api-bindings/src/graphql/utils/index.ts

```diff
@@ -1,6 +1,7 @@
 // generic utils
 export * from './amount';
 export * from './errors';
+export * from './isValidHandle';
```

Comment:
on the other note which is not a part of this PR, wondering about the way to share the `isValidHandle` between `api-bindings` and this new package with just core api wrappers to avoid repeating the same regexes twice

Reply:
good point... I put it here cause it's a validation very related to the API. Not sure if the low-level client will need such helper.... fair to assume so though. Let's see when we get there.